src/components/ThreadEnvelope.vue

```diff
@@ -198,6 +199,9 @@ export default {
 		cleanSubject() {
 			return this.envelope.subject.replace(/((?:[\t ]*(?:R|RE|F|FW|FWD):[\t ]*)*)/i, '')
 		},
+		centeredSender() {
+			  return (!this.hasChangedSubject || this.cleanSubject.length === 0) && this.tags.length === 0
+				 },
```

Comment:
something looks odd here

Reply:
yes? i dont see it :(

this was the only way how i could make it work for all the scenarios. Am i missing something, or the way how i wrote the conditions is wrong?

Reply:
The indentation